AppCenter/AppCenterTests/MSLogDBStorageTests.m

```diff
@@ -146,38 +143,37 @@ - (void)testLoadCommonSchemaLogsWhilePendingBatchesWithSpecificTargetKeys {
   MSCommonSchemaLog *log1 = [MSCommonSchemaLog new];
   [log1 addTransmissionTargetToken:@"1-t"];
   log1.iKey = @"o:1";
-  [self.sut saveLog:log1 withGroupId:kMSTestGroupId];
+  [self.sut saveLog:log1 withGroupId:kMSTestGroupId critical:nil];
```

Comment:
I can't tell what the intended behavior is here, is using `critical:nil` supposed to have the same behavior as using `critical:NO`?  Did you choose to use the former because the test is criticality-agnostic?  I would prefer `criticality:NO` if it doesn't matter since the behavior isn't obvious.

Reply:
Let's avoid using nil indeed, I find it very confusing too.